python/lsst/dax/imgserv/templates/uws_job_result.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I was just looking at the code again and I'm not sure where this is used, like I don't see the template being applied somewhere in the code, but this looks like a new file?

Reply:
Sorry this meant to be on uws_job_descriptor.xml

Reply:
Will replace the plain text response used during dev testing with this one.